KalebIrwin-Power-signoff.md

```diff
@@ -0,0 +1,42 @@
+## Function of the subsystem 
+The subsystem will provide the microcontroller and sensor interface with power. There will be two configurations designed. The first configuration will be designed to operate with input from an electrical outlet and the second configuration will be powered by a battery.
+
+## Constraints 
+1. Provide 1.96W of power 
+   a. *1.75W to Arduino Mega 
+   b. 210mW for three high current draw sensors simultaneously
+Constraints on system for AC operation
+2. Function on 120V 60Hz AC input
+3. Power the Arduino Mega with 5V USB input
+4. **Provide 5V and 3V DC for sensor interface 
+Constraints on system for battery operation
+5. Power the Arduino Mega with 7-12V or 5V through the USB connector 
+6. **Provide 5V and 3V for sensor interface 
+7. Battery capacity greater than 18.816 Ah 
+   Calculated with (maximum power draw*minimum continuous operation time)/battery voltage =(1.96W * 48h)/ 5V = 18.816 Ah  
+8. Must make use of a solar panel to extend the operational time without maintenance 
+
+## Schematic 
+<img width="866" alt="BuildableAC" src="https://user-images.githubusercontent.com/118766525/216235986-c4a7bbef-dcb8-4c07-8018-090d6612bf4e.png">
+<img width="627" alt="BuildableDC" src="https://user-images.githubusercontent.com/118766525/216236032-495901d5-41aa-42f0-b8a0-075caf56226f.png">
+
+## Analysis 
```

Comment:
Need to show that each device on BOM satisfies the requirements. For example very little is said about the dc-dc converter, so its specifications need to be analyzed to show it meets the requirements. Is the solar charger suitable for permanent mounting outside the enclosure, e.g. weatherproof, size, provisions for mounting it? Also see previous comment about amp-hours.